### DISCUSSION TITLE
deployments & revisionHistoryLimit

### DISCUSSION THREAD

**Discussion by kfox1111** [aquasecurity/starboard#668] [Development]
We've got a lot of replicasets on some of our clusters. deployment is defaulting to revisionHistoryLimit = 10 I think these days. Which means there could be an awful lot of scans happening on older replicasets from deployments that have long histories. Starboard should probably ignore all replicasets where replicas == 0.

**Comment by danielpacak** [comment]
We are continuing this discussion under #858
